app/_locales/en/messages.json

```diff
@@ -3256,50 +3256,85 @@
   "onboardingMetametricsAgree": {
     "message": "I agree"
   },
-  "onboardingMetametricsAllowOptOut": {
+  "onboardingMetametricsAllowOptOut_legacy": {
```

Comment:
Could you update these keys to follow our message naming conventions? camelCase, rather than underscores

Reply:
@jonybur This is still relevant.